R/test.R

```diff
@@ -46,14 +59,20 @@ use_testthat_impl <- function(edition = NULL) {
 }
 
 check_edition <- function(edition = NULL) {
-  testthat_version <- utils::packageVersion("testthat")[[1, 1]]
+  version <- utils::packageVersion("testthat")[[1, c(1, 2)]]
+  if (version[[2]] == "99") {
```

Comment:
Should we insert a TODO to simplify this once testthat releases?

Reply:
I think it would be slightly better to introduce a `major_version()` helper that wrapped this up, as it's a pattern we might need elsewhere.